inventory.lua

```diff
@@ -83,63 +104,109 @@ minetest.register_node("digilines:chest", {
 			return not pipeworks.connects.facingFront(i,param2)
 		end,
 		input_inventory = "main",
-		can_insert = function(pos, _, stack)
-			return can_insert(pos, stack)
+		can_insert = function(pos, _, stack, vel)
+			local ret = minetest.get_meta(pos):get_inventory():room_for_item("main", stack)
+			if not ret then
+				-- The stack cannot be accepted. It will never be passed to
+				-- insert_object, but it should be reported as a toverflow.
+				-- Here, vel = direction item is moving, which is into side.
+				local side = vector.new(-vel.x, -vel.y, -vel.z)
```

Comment:
`vector.multiply(vel, -1)`

Reply:
Fixed in e9056bc72663acc7a2945eeeb0f3ad33b42e040d.